packages/core/src/styles/themes/themes.stories.ts

```diff
@@ -0,0 +1,381 @@
+  } else if (typeof token.value === 'number') {
+    token.formattedValue = html`<span cds-text="code">${(token.value / 20).toFixed(2)}rem (${token.value}px)</span>`;
+  } else {
+    token.formattedValue = html`<span cds-text="code">${token.value}</span>`;
+  }
+
+  return token;
+}
+
+export const lightTheme = () => {
+  return html`
+    <section cds-layout="vertical gap:lg">
+      <cds-input layout="vertical" control-width="shrink">
+        <label>text label</label>
+        <input placeholder="placeholder text" />
+        <cds-control-message>message</cds-control-message>
+      </cds-input>
+
+      <cds-checkbox-group layout="vertical-inline">
+        <label>checkbox group label</label>
+        <cds-checkbox>
+          <label>checkbox 1</label>
+          <input type="checkbox" checked />
+        </cds-checkbox>
+
+        <cds-checkbox>
+          <label>checkbox 2</label>
+          <input type="checkbox" />
+        </cds-checkbox>
+
+        <cds-checkbox>
+          <label>checkbox 3</label>
+          <input type="checkbox" />
+        </cds-checkbox>
+      </cds-checkbox-group>
+
+      <cds-button><cds-icon shape="user"></cds-icon>primary<cds-badge>10</cds-badge></cds-button>
+
+      <div cds-layout="horizontal gap:lg align:vertical-center">
+        <cds-progress-circle size="xl"></cds-progress-circle>
+
+        <cds-tag status="info">
+          <cds-icon shape="info-standard"></cds-icon>Info <cds-badge status="info">10</cds-badge>
+        </cds-tag>
+
+        <cds-icon shape="user" size="lg" badge="info"></cds-icon>
+      </div>
+    </section>
+  `;
+};
+
+export const darkTheme = () => {
+  return html`
+    <section cds-theme="dark" cds-layout="vertical gap:lg">
+      <cds-input layout="vertical" control-width="shrink">
+        <label>text label</label>
+        <input placeholder="placeholder text" />
+        <cds-control-message>message</cds-control-message>
+      </cds-input>
+
+      <cds-checkbox-group layout="vertical-inline">
+        <label>checkbox group label</label>
+        <cds-checkbox>
+          <label>checkbox 1</label>
+          <input type="checkbox" checked />
+        </cds-checkbox>
+
+        <cds-checkbox>
+          <label>checkbox 2</label>
+          <input type="checkbox" />
+        </cds-checkbox>
+
+        <cds-checkbox>
+          <label>checkbox 3</label>
+          <input type="checkbox" />
+        </cds-checkbox>
+      </cds-checkbox-group>
+
+      <cds-button><cds-icon shape="user"></cds-icon>primary<cds-badge>10</cds-badge></cds-button>
+
+      <div cds-layout="horizontal gap:lg align:vertical-center">
+        <cds-progress-circle size="xl"></cds-progress-circle>
+
+        <cds-tag status="info">
+          <cds-icon shape="info-standard"></cds-icon>Info <cds-badge status="info">10</cds-badge>
+        </cds-tag>
+
+        <cds-icon shape="user" size="lg" badge="info"></cds-icon>
+      </div>
+    </section>
+  `;
+};
+
+export const dynamicTheme = () => {
```

Comment:
This demo shows how the alias tokens can provide a powerful and flexible theme system that will reflect across all components.